x-pack/plugins/maps/public/shared/layers/styles/components/vector/legend/style_property_legend_row.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import { styleOptionShapes, rangeShape } from '../style_option_shapes';
+import { VectorStyle } from '../../../vector_style';
+import { ColorGradient } from '../../../../../icons/color_gradient';
+import { FillableCircle } from '../../../../../icons/additional_layer_icons';
+import { getVectorStyleLabel } from '../get_vector_style_label';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiSpacer,
+  EuiToolTip,
+  EuiHorizontalRule,
+} from '@elastic/eui';
+
+function getLineWidthIcons() {
+  function getStyle(strokeWidth) {
+    return {
+      stroke: 'grey',
+      strokeWidth,
+      fill: 'none',
+      width: '12px',
+    };
+  }
+
+  return [
+    <FillableCircle style={getStyle('1px')}/>,
+    <FillableCircle style={getStyle('2px')}/>,
+    <FillableCircle style={getStyle('3px')}/>,
+  ];
+}
+
+function getSymbolSizeIcons() {
+  function getStyle(width) {
```

Comment:
this is weird. either move `getStyle` out of this function, or just use mixin pattern to get those pixel-defaults applied to a new style-object